samples/word/04-range/split-words-of-first-paragraph.yaml

```diff
@@ -0,0 +1,161 @@
+id: word-basics-split-words-of-first-paragraph-
+name: Range manipulation
+description: 'Show how to split a paragraph into word ranges and then traverses all the ranges to format each word, producing a "karaoke" effect.'
+author: Microsoft
+host: WORD
+api_set:
+    WordApi: 1.1
+script:
+    content: |-
+        $("#setup").click(setup);
+        $("#highlight").click(highlightWords);
+
+        function highlightWords() {
```

Comment:
This sample would seriously benefit from TypeScript-ification.  And you'd actually be able to get rid of the entirety of the nesting